src/components/widgets/Two.vue

```diff
@@ -1,3 +1,93 @@
 <template>
-	<div></div>
-</template>
\ No newline at end of file
+  <div class="maps">
+    <Search
+      placeholder="Search for a location"
+      v-model:searchText="searchText"
+      :search="searchLocation"
+    />
+    <div id="map" />
+  </div>
+</template>
+
+<script>
+import Search from "../common/Search.vue";
+import mapboxgl from "mapbox-gl";
+import "mapbox-gl/dist/mapbox-gl.css";
+// import MapboxDirections from '@mapbox/mapbox-gl-directions'
+import { onMounted } from "vue";
+
+export default {
+  name: "Two",
+  data: () => ({
+    searchText: "",
+    map: '',
+    accessToken: import.meta.env.VITE_MAPS_API_KEY,
+    mapStyle: "mapbox://styles/mapbox/streets-v11",
+    zoom: 14,
+    center:[-0.0613707, 51.5348032],
+    markers:[],
+  }),
+  components: {
+    Search,
+  },
+  methods: {
+    async searchLocation() {
+      this.removeMarkers()
+
+      const address = this.searchText.split(' ').join('+');
+      const response = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${address}&key=${import.meta.env.VITE_GEOCODING_API_KEY}`)
+      const result = await response.json();
+      const center_lng = result.results[0].geometry.location.lng
```

Comment:
A few semicolons could be added to keep consistency.

Reply:
Nice catch -- Fixed :)